lib/openscap_parser/profile.rb

```diff
@@ -1,6 +1,20 @@
 module OpenscapParser
   class Profile
-    attr_acessor :id, :title, :description
+    def initialize(profile_xml: nil)
+      @profile_xml = profile_xml
```

Comment:
I like this architecture of passing in the Nokogiri::Element like we do for `OpenscapParser::Rule` objects